CampaignConfig/mdc2020_crymix.cfg

```diff
@@ -1,28 +1,27 @@
 [global]
 includes = /mu2e/app/users/sophie/ProductionPush/Production/CampaignConfig/mdc2020_prolog.cfg
-desc = MDC2020x
+desc = MDC2020y
 run_number = 1205
 release = MDC2020
-release_v_o = x
-db_version = v1_0
+release_v_o = y
+db_version = v1_1
 db_purpose = perfect
 digitype = OnSpill
 bfield = "Offline/Mu2eG4/geom/bfgeom_no_field.txt"
-njobs=10000
 
-njobs=10000
 
 [stage_crydigi_fcl]
-global.stage_name = CosmicCRYExtractedDigi
+global.stage_name = CosmicCRY%(digitype)sDigi
```

Comment:
This is pure digitization (runs gen_Digitise.sh)

Reply:
then  why does the file name contain crymix?  Maybe it should be renamed?

Reply:
Yes, this file does both digi, reco and mixing. But I'd like to have this campaign as is in Production for the record since its what worked for our last CRY job